src/recipe_dictionary.cpp

```diff
@@ -33,9 +33,57 @@ std::vector<const recipe *> recipe_subset::search( const std::string &txt ) cons
 {
     std::vector<const recipe *> res;
 
+    size_t split;
+    split = txt.find( ":" );
+    std::string key = "", value;
+    if( split != std::string::npos ) {
+        key = txt.substr( 0, split );
```

Comment:
Fair enough. I wanted the search to tolerate spurious spaces. I trust our users might accidentally do that.